### DISCUSSION TITLE
Dokploy and private container images repository (distribution)

### DISCUSSION THREAD

**Discussion by cadilhac** [Dokploy/dokploy#881] [Q&A]
I have set up my own private docker registry on my local machine with https://distribution.github.io/distribution. It is protected with TLS and a self-signed certificate. There is no username and password.
I have no idea how to make my repository known to dokploy... Is it even feasible?
Thanks

**Comment by Siumauricio** [comment]
I have never used distribution, but we offer 2 ways to connect to a registry, using registry from the settings section, and also if you use applications you can specify a registry url
